src/include/mpir_request.h

```diff
@@ -367,6 +367,20 @@ MPL_STATIC_INLINE_PREFIX int MPIR_Request_completion_processing_fastpath(MPI_Req
 int MPIR_Request_completion_processing(MPIR_Request *, MPI_Status *, int *);
 int MPIR_Request_get_error(MPIR_Request *);
 
+extern int MPID_Request_is_anysource(MPIR_Request *);
+extern int MPID_Comm_AS_enabled(MPIR_Comm *);
+extern int MPIR_CVAR_ENABLE_FT;
+
+/* The following routines are helper to handle ULFM. */
```

Comment:
I think this comment would be more useful if it said what it did and not that it's a ULFM-helper. I guess it could say that it's a ULFM helper too.